applications/vanilla/modules/class.discussionssortfiltermodule.php

```diff
@@ -128,18 +128,18 @@ protected function getSortData() {
                 }
             }
             $key = val('key', $sort);
-            $queryString = DiscussionModel::getSortFilterQueryString($this->selectedSort, $this->selectedFilters, $key);
+            $queryString = val('key', $sort) !== 'none' ? DiscussionModel::getSortFilterQueryString($this->selectedSort, $this->selectedFilters, $key) : '';
```

Comment:
"none" should be replaced with `DiscussionModel::EMPTY_FILTER_KEY` (the constant for "none").